src/Reader.js

```diff
@@ -76,6 +76,57 @@ function addPropWithTextContent(node, obj, prop, trimText = false) {
   }
 }
 
+function addFilterExpressionProp(node, obj, prop, skipEmptyNodes = true) {
```

Comment:
What is the purpose of this function? 
Link to: http://schemas.opengis.net/filter/1.1.0/expr.xsd for documentation?

Reply:
I've added documentation for addFilterExpressionProp in b127dd4